ansible/playbook.yaml

```
@@ -0,0 +1,71 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Please input Trello Api Key
+      private: yes
+
+    - name: trello_api_token
+      prompt: Please input Trello Api Token
+      private: yes
+
+  tasks:
+  - name: Install the latest version of git
+    become: yes
+    yum:
+      name: git
+      state: latest
+
+  - name: Install the latest version of python
+    become: yes
+    yum:
+      name: python3
+      state: latest

```

More a "in case you're interested" than a recommendation, but worth noting we could combine these:
```suggestion
  - name: Install the latest version of git & Python
    become: yes
    yum:
      name:
        - git
        - python3
      state: latest
```